default/python/AI/ColonisationAI.py

```diff
@@ -395,10 +387,10 @@ def survey_universe():
         table_name="Empire species roster"
     )
 
-    for spec_name in empire_species:
+    for spec_name, planets in state.get_species_planets().items():
```

Comment:
These are planet ids not planet objects, so should be named ```pids``` or ```planet_ids``` but probably not planets.